### ISSUE TITLE
[Manager] pipeline setting up connector using Mysql jdbc cannot proceed, giving "No suitable driver found for  URL" warning

### ISSUE THREAD

**Issue by alan791205** [oharastream/ohara#1102]
As title,
I created a broker, zookeeper, and a connect cluster with a "mysql-connector-java-8.0.12.jar".
Then I created a pipeline and added a jdbc source connector, after I filled in informations and tried to get tables from this button
![擷取選取區域_073](https://user-images.githubusercontent.com/8744475/57447492-ca33f700-7289-11e9-9559-962779f7e5b6.png)

and it will pass this message: "No suitable driver found for URL" but the test will pass.
![擷取選取區域_074](https://user-images.githubusercontent.com/8744475/57447647-1f700880-728a-11e9-84e4-67ca732c5b5f.png)

then I cannot proceed because of cannot get table names.

Using CentOS
```
~ # cat /etc/redhat-release                                                                                                                                                      
CentOS Linux release 7.5.1804 (Core) 
------------------------------------------------------------
```
Chrome
版本 70.0.3538.77 (正式版本) (64 位元)

**Comment by chia7712** [comment]
Oh... we forgot to fix this for 0.4.

In the future, all connector pages are generated by connector’s setting(see docs/custom_connector.md) and the helper functions, such as getTable, are removed.

There are two solutions.
1. remove the button from UI, and requires user to enter the table name manually 
2. Enable user to put driver when starting configurator so it works for MySQL

I prefer first solution since it should be a small patch. 

@konekoya @eechih @wu87988622 any suggestions?

**Comment by eechih** [comment]
目前我也傾向第 1 個解法。長遠之計，需要討論如何讓 metadata 可以描述更多的 UI。

**Comment by konekoya** [comment]
> 1. remove the button from UI, and requires user to enter the table name manually

This does solve the problem. But the current UI which provides a dropdown for users to choose from is generally a better UI and easier to use for users. Is there any other ways that we can achieve this with metadata API...

**Comment by konekoya** [comment]
> 目前我也傾向第 1 個解法。長遠之計，需要討論如何讓 metadata 可以描述更多的 UI。

+1 as a temp fix

**Comment by chia7712** [comment]
> But the current UI which provides a dropdown for users to choose from is generally a better UI and easier to use for users. Is there any other ways that we can achieve same UI with metadata API...

As I mentioned before, this solution requires user to deploy the driver to Configurator for database. And it introduces many extra “know how” to user and it makes unfriendly use.

To resolve the gap, the backend needs to address many tasks.
1. Introduce jdbc.Table type
1. A official helper connector used to fetch the table information 
1. Establish connection between configurator and worker cluster

The side-effect is a huge latency but user don’t worry about the jar deployment to configurator.

**Comment by konekoya** [comment]
Thanks for the detailed elaboration on this topic. This does sound like a lot of work need to be done before we can have the same UI with the new metadata API. I'll go with the solution one, so users can at least use jdbc connector for now :)

@eechih @wu87988622 thoughts before I file another issue to address this

**Comment by chia7712** [comment]
> I'll go with the solution one, so users can at least use jdbc connector for now :)

+1

BTW, #1107 is follow-up of resolving this issue totally.

**Comment by eechih** [comment]
> I'll go with the solution one, so users can at least use jdbc connector for now :)

+1

**Comment by konekoya** [comment]
A temp UI fix will be tracked in #1109 

@chia7712 Can we close this issue and in favor of #1107 and #1109

**Comment by chia7712** [comment]
> Can we close this issue and in favor of #1107 and #1109

Please attach the conclusion to the description and then close this issue. Thanks!

**Comment by konekoya** [comment]
> Please attach the conclusion to the description and then close this issue. Thanks!

Added :)

**Comment by konekoya** [comment]
Thanks for raising the issue. Please follow #1107 and #1109 for updates :)
